core/templates/pages/contributor-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -247,6 +253,43 @@ export class ContributionOpportunitiesBackendApiService {
       return [];
     }
   }
+
+  async savePreferredTranslationLanguageAsync(
+      languageCode: string
+  ): Promise<void> {
+    return this.userService.getUserInfoAsync().then(
+      (userInfo) => {
+        if (userInfo?.isLoggedIn()) {
```

Comment:
Why can be `userInfo` `undefined`?

Reply:
It can not be undefined. I have updated this and removed the `?`.